ci_framework/hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -122,6 +124,26 @@
                     value: {{ crc_ci_bootstrap_networks_out.compute['internal-api'].mtu | default(1500) }}
               {% endif %}
 
+              {% if 'compute1' in crc_ci_bootstrap_networks_out %}
```

Comment:
Uhmm this thing will scale better if done with a loop

Reply:
pls take a look on newer commit.